frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx

```diff
@@ -481,8 +482,14 @@ const CreateBackingStoreForm: React.FC<CreateBackingStoreFormProps> = withHandle
     providerDataReducer,
     initialState,
   );
+  const [disabled, setDisabled] = React.useState(true);
 
-  const { cancel, close, inProgress, errorMessage, handlePromise, isPage } = props;
+  React.useEffect(() => {
+    if (bsName.length > 0 && providerDataState.target.length > 0) setDisabled(false);
```

Comment:
dont you need to call trim() to make sure you wont accept just space like ` ` ? 

You can also make it shorter by calling just `setDisabled(!(bsName.length > 0 && providerDataState.target.length > 0))`

Reply:
ack